llvm/test/MC/AArch64/SVE2/directive-arch-negative.s

```diff
@@ -29,3 +29,21 @@ rax1 z0.d, z0.d, z0.d
 bgrp z21.s, z10.s, z21.s
 // CHECK: error: instruction requires: sve2-bitperm
 // CHECK-NEXT: bgrp z21.s, z10.s, z21.s
+
+.arch armv9-a+sve2+f8f16mm
```

Comment:
I believe +sve2 is a default of armv9-a, so it might not be necessary in these.

Reply:
Done